src/index.js

```diff
@@ -0,0 +1,20 @@
+const API_KEY = "aaebd4178fcf227a3349fb75ef211df7";
```

Comment:
A good practice is to move sensitive information to separate file, such as config.js and import them here to use the API_KEY and API_URL.

Reply:
Fixed